lib/Native/NativeEncoder.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types = 1);
```

Comment:
Please drop this, we don't use `strict_types` in our libraries.

Reply:
oki